.github/workflows/CI.yml

```diff
@@ -75,7 +72,7 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        python -m pip install mpi4py pytest pytest-mpi fortranformat
+        python -m pip install mpi4py pytest pytest-mpi fortranformat anesthetic
```

Comment:
We might not actually need pytest-mpi (which allows us to turn tests on or off depending on whether the test is mpirun) - do you see us ever wanting to test something on serial mode only?